packages/@okta/vuepress-site/docs/guides/deploy-your-app/main/javascript/deployment.md

```diff
@@ -1,6 +1,6 @@
-## Deploying a JavaScript app
+## Deploy a JavaScript app
 
-JavaScript applications, when built for production, are just a bunch of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go!
+JavaScript apps built for production are a collection of static files. You can deploy them to any web server, configure that web server to be SPA-aware, and you should be good to go.
```

Comment:
```suggestion
JavaScript apps built for production are a collection of static files. You can deploy them to any web server, and then configure that web server to be SPA-aware.
```

Reply:
not sure if something like "and then you're ready to use them" is a reasonable replacement for "should be good to go" but if not I think it's fine to just cut. (not sure how well it translates)